src/GridList/GridTile.js

```diff
@@ -0,0 +1,154 @@
+// @flow weak
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { createStyleSheet } from 'jss-theme-reactor';
+import customPropTypes from '../utils/customPropTypes';
+
+export const styleSheet = createStyleSheet('MuiGridTile', (theme) => {
+  return {
+    root: {
+      position: 'relative',
+      display: 'block',
+      height: '100%',
+      overflow: 'hidden',
+      fontFamily: theme.typography.fontFamily,
+    },
+    childImg: {
+      height: '100%',
+      transform: 'translateX(-50%)',
+      position: 'relative',
+      left: '50%',
+      maxWidth: 'none', // Reset to initial value to override docs-site App-frame.
+    },
+  };
+});
+
+/**
+ * ```sx
+ * <GridTile>
+ *   <img src="image.jpg" />
+ *   <GridTileTitlebar title="GridTile" />
+ * </GridTile>
+ * ```
+ */
+export default class GridTile extends Component {
+  static propTypes = {
+    /**
+     * Theoretically you can pass any node as children, but the main use case is to pass an img,
+     * in which case GridTile takes care of making the image "cover" available space
+     * (similar to `background-size: cover` or to `object-fit: cover`).
+     */
+    children: PropTypes.node.isRequired,
+    /**
+     * The CSS `className` of the root element.
+     */
+    className: PropTypes.string,
+    /**
+     * Width of the tile in number of grid cells.
+     */
+    cols: PropTypes.number,
+    /**
+     * Either a string used as tag name for the tile root element, or a ReactElement.
+     * This is useful when you have, for example, a custom implementation of a navigation
+     * link (that knows about your routes) and you want to use it as the primary tile action.
+     * In case you pass a ReactElement, please ensure that it passes all props,
+     * accepts styles overrides and render it's children.
+     */
+    containerElement: PropTypes.oneOfType([
```

Comment:
We use the `component` property on the rest of the code base to address that same issue. Sure we could rethink about it but I would rather be consistent. Notice that the pattern is different. With `containerElement` you provide an element with some properties, with `component` you provide a component and apply the properties on the `GridTile`.
```js
  /**
   * The component used for the root node.
   * Either a string to use a DOM element or a component.
   */
  component: PropTypes.oneOfType([
    PropTypes.string,
    PropTypes.func,
  ]),
```